dhall/src/Dhall/Util.hs

```diff
@@ -12,6 +12,7 @@ module Dhall.Util
     , Output(..)
     , getExpression
     , getExpressionAndHeader
+    , Header(..)
```

Comment:
What's the motivation for this re-export? I'm wondering whether `Header` should live in `Dhall.Syntax`… :thinking:

Reply:
I didn't mean to commit that. 😅 I used to have the `Header` newtype defined in that module. I'll remove the re-export!

Reply:
Ah wait no, I was mixing up with a different module. I re-exported it here, because several module use `Dhall.Util` to use `getExpressionAndHeader`, so it made sense to re-export `Header` with it.

What's the rational for what goes into `Dhall.Syntax`?

Reply:
> What's the rationale for what goes into `Dhall.Syntax`?

I wrote

https://github.com/dhall-lang/dhall-haskell/blob/ad443cd6851af215111e21d8ac92a9fb3cda7403/dhall/src/Dhall/Syntax.hs#L14-L19

But maybe `Headers` isn't a "core syntax type".

Let's leave things as they are now then. :+1: